backend/LeetWars.Notifier/LeetWars.Notifier.WebAPI/Services/NotificationConsumerService.cs

```diff
@@ -28,12 +28,14 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
                 var notificationDto = JsonSerializer.Deserialize<NewNotificationDto>(message);
 
-                if (notificationDto != null)
+                if(notificationDto is null)
                 {
-                    await SendNotificationAsync(notificationDto);
+                    await Task.CompletedTask;
```

Comment:
why await not return?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/93576084/4b956f72-5355-4b19-92c3-591530f3fdf9)